cloud/src/meta-service/txn_kv.cpp

```diff
@@ -527,6 +528,17 @@ void Transaction::remove(std::string_view begin, std::string_view end) {
 }
 
 TxnErrorCode Transaction::commit() {
+    if (config::txn_commit_fault_probability > 0.0) [[unlikely]] {
```

Comment:
use injection point instead 
see common/cpp/sync_point.h TEST_INJECTION_POINT_RETURN_WITH_VALUE
and cloud/src/meta-service/injection_point_http.cpp